### PR TITLE
Add button get access functionality & Add footer to facilitators page

### PR DESCRIPTION
**Pull request by sharonm6**

### CODE REVIEW

**Comment by aboutedsight** [comment]
Please provide a screenshot of what changed on the PR itself :)

File: facilitators.html [hunk 1]
```diff
@@ -135,7 +135,113 @@ <h2 class="new-color-text h2 fs-1 mb-4">How to Set Up Edsight in Four Steps</h2>
             </div>
         </div>
     </section>
-
+    <!-- Footer-->
+    <footer class="bg-black text-center py-5">
+        <div class="container px-5">
+            <div class="text-white-50 small">
+                <div class="mb-2">&copy; Your Website 2022. All Rights Reserved.</div>
```
Comment (aboutedsight) [inline facilitators.html]: Instead of "Your Website 2022", can you change this to "UCI Design and Partnership Lab, 2022" ?

**Review by aboutedsight — APPROVED** [review]
Looks good so far! Please make the change to "Your Website" on all 3 pages before completing this pull request (no need for me to review again)

**Comment by sharonm6** [comment]
Screen Shot 2022-11-22 at 1 29 33 PM

**Pull request merged**